api/v1/sriovnetworknodepolicy_types.go

```diff
@@ -57,6 +57,8 @@ type SriovNetworkNodePolicySpec struct {
 	// +kubebuilder:validation:Enum=virtio
 	// VDPA device type. Allowed value "virtio"
 	VdpaType string `json:"vdpaType,omitempty"`
+	// Exclude device's NUMA node when advertising this resource. Default to false.
```

Comment:
nit:  Exclude device's NUMA node when advertising this resource by sriov network device plugin

Reply:
Looks good, thanks